charts/incubator/alist/Chart.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: alist
+version: 0.0.1
+appVersion: 1.0.0
+description: alist App for TrueNAS SCALE
+type: application
+deprecated: false
+home: https://github.com/truecharts/apps/tree/master/charts/stable/alist
+icon: https://u.pigax.cn/gh/alist-org/assets@main/logo.svg
```

Comment:
Please put a png icon in the docs/_static/appicons and link it correctly (check other apps)